app/src/main/java/com/cp/tasky/auth/login/presentation/LoginScreen.kt

```diff
@@ -207,6 +212,8 @@ fun LoginScreenPreview() {
     LoginScreen(
         viewState = LoginScreenState(),
         onEvents = { LoginScreenEvent.SetEmail("") },
-        networkState = Result.Idle
+        networkState = Result.Idle,
```

Comment:
Instead of working with such Idle results, I'd make this field nullable instead, since that's exactly what nulls are used for (indicating that a value doesn't exist). That way you communicate by design that there's a chance that this value is not yet existing.